### PR TITLE
Fix windows path bug, add jest as test option

### PR DESCRIPTION
**Pull request by flyfishMT**
PR for issues #2 and #16

### CODE REVIEW

**Comment by snaerth** [comment]
Fantastic I will take a look at it very soon

**Comment by flyfishMT** [comment]
I have a subsequent PR I'm working on that makes this a little cleaner, but I am having some issues with the commander lib.  It changes the options to --test [enzyme|jest|none]. Let me know if this doesn't make sense to you as I am new to react. And it adds --testfolder [glob pattern]. If testfolder option is given it searches for this folder and writes tests to here rather than component folder.  (I want my tests to go into /src/__tests__).  But so far to do this I've had to pass component(s) to create with as an option (--components one two three). So I'll have to figure that out to not break backwards compatibility.  Anyways let me know if you are interested in these ideas.  This library is really helpful to me thank you! - With those 2 little changes it will be perfect for my use case.

File: lib/utils/componentsHelpers.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const isWin = process.platform === 'wind32';
+const isWin = process.platform === 'win32';
```
Comment (snaerth) [inline lib/utils/componentsHelpers.js]: Does this fix the windows error

Comment (flyfishMT) [reply lib/utils/componentsHelpers.js]: yes

**Pull request closed without merging**